### ISSUE TITLE
Keep language option when restart

### ISSUE THREAD

**Issue by JeromeLeCoz** [BaRatin-tools/BaRatinAGE#85]
It would be convenient not to have to select the language each time we launch the software.

**Comment by IvanHeriver** [comment]
indeed, I though that was implemented but it might have been lost along the way. Currently, the system default local is used each time at start up. 

I think this issue is closely related to issue #80

**Comment by IvanHeriver** [comment]
fixed in v3.1
